### PR TITLE
User story18

### PR DESCRIPTION
**Pull request by sagemlee**
-Add list of Pets that have applications 

-Update favorites index spec 
-Update favorites index view 
-All tests passing

### CODE REVIEW

File: app/controllers/favorites_controller.rb [hunk 1]
```diff
@@ -1,32 +1,33 @@
 class FavoritesController < ApplicationController
   include ActionView::Helpers::TextHelper
-  
+
   def index
     pet_ids = favorite.contents.keys
-    @pets = Pet.find(pet_ids)
+    @favorite_pets = Pet.find(pet_ids)
```
Comment (sagemlee) [inline app/controllers/favorites_controller.rb:6]: I feel like this is something we can look into refactoring later

Comment (dborski) [reply app/controllers/favorites_controller.rb:6]: Wasn't one of these supposed to be a Pet.all?

Comment (sagemlee) [reply app/controllers/favorites_controller.rb:6]: is what you are looking for on line 7?

File: spec/features/favorites/see_favorite_spec.rb [hunk 2]
```diff
@@ -28,10 +28,50 @@
...
+        expect(page).to have_link("#{pet3.name}")
+        click_link("#{pet3.name}")
+        expect(current_path).to eq("/pets/#{pet3.id}")
+      end
+
+      visit '/favorites'
+
+      within("#pets_with_applications") do
+        expect(page).to_not have_link("#{pet1.name}")
+      end
```
Comment (dborski) [inline spec/features/favorites/see_favorite_spec.rb:73]: Nice job catching that we need to visit favorites again for that test to work

**Pull request merged**